mod/state-transition/pkg/core/state_processor_staking.go

```diff
@@ -193,18 +199,18 @@ func (sp *StateProcessor[
 
 	// TODO: This is a bug that lives on bArtio. Delete this eventually.
 	if sp.cs.DepositEth1ChainID() == bArtioChainID {
-		if err := st.AddValidatorBartio(val); err != nil {
-			return err
-		}
```

Comment:
`AddValidatorBartio` already implicitly set the balance, so if we don't return here, we are setting the balance twice.
@itsdevbear was this expected?
If not, the change I propose fixes this
If yes, happy to revert and fix this in the UT